queries.sql

```diff
@@ -1,3 +1,11 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE animal_name LIKE '%mon';
+SELECT animal_name FROM animals WHERE date_of_birth BETWEEN '2016/02/01' AND '2019/01/04';
```

Comment:
- Hey, since we want to get the list of the name of all animals born between 2016 and 2019, the `date_of_birth` should be between `'2016/01/01' AND  '2019/31/12';` to get the correct query. Please, update this.